scripts/constants.py

```diff
@@ -15,15 +15,15 @@
 import sys
 
 # Kubernetes branch to get the OpenAPI spec from.
-KUBERNETES_BRANCH = "release-1.5"
+KUBERNETES_BRANCH = "b0b7a323cc5a4a2019b2e9520c21c7830b7f708e"
```

Comment:
I don't have a strong opinion about this yet (so I am changing it back to release-1.6) but need need to discuss this. A branch may change (patches) and we should somehow keep track of which spec we created client from (until we have a consistent versioning in spec itself).